spec/requests/health_monitor_spec.rb

```diff
@@ -11,9 +11,6 @@
   describe "GET /health" do
     it "has a success response" do
```

Comment:
Can you update the test to make it clear we expect success even when a non-critical service is failing?